server/src/main/scala/manifests/passengers/PassengerInfo.scala

```diff
@@ -5,18 +5,27 @@ import passengersplits.parsing.VoyageManifestParser.VoyageManifest
 import queueus.B5JPlusWithTransitTypeAllocator
 import uk.gov.homeoffice.drt.Nationality
 import uk.gov.homeoffice.drt.ports.{PaxAge, PaxType}
+import uk.gov.homeoffice.drt.time.{SDate, SDateLike}
 
 object PassengerInfo {
 
-  val ageRanges: List[AgeRange] = List(
-    AgeRange(0, 9),
-    AgeRange(10, 24),
-    AgeRange(25, 49),
-    AgeRange(50, 65),
-    AgeRange(65),
-  )
+  def ageRanges(scheduled: Option[SDateLike]): List[AgeRange] = {
+    val childEligibility = scheduled match {
+      case Some(date) if date < SDate("2023-07-25T01:00:01") =>
```

Comment:
This is also the wrong date, and also has a random hour and 100 milliseconds on it!
Maybe best to keep one value for this date somewhere and refer to it when needed